test/v3nametest.c

```diff
@@ -288,6 +288,9 @@ static int run_cert(X509 *crt, const char *nameincert,
         char *name = OPENSSL_malloc(namelen + 1);
         int match, ret;
 
+        if (name == NULL) {
+            return 1;
+        }
```

Comment:
This is wrong.  Returning success on a memory allocation failure is bad.  Try:
```c
if (!TEST_ptr(name))
    return 0;
```

Reply:
Thanks, I have submitted a fixup.